stories/accordion/index.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
+// @ts-ignore TODO-TS: remove this when updating storybook version since newest includes types
 import { storiesOf } from '@storybook/react';
-import { withDocs } from 'storybook-readme';
+// @ts-ignore TODO-TS: remove this when updating storybook version since newest includes types
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import { withDocs } from 'storybook-readme';
 import { action } from '@storybook/addon-actions';
 import { css } from '@emotion/core';
-import AccordionReadme from 'docs/accordion';
-import { Accordion, Typography } from 'src/shared-components';
-import { SPACER } from 'src/constants';
 
+// @ts-ignore
```

Comment:
I don't think we should merge this in before resolving the issue. We'll want to update how we're handling resolving paths. It is probably just because it is a markdown file and we didn't add that as an extension, in addition to `.ts` and `.tsx`, somewhere.

Reply:
solved. added as a custom module. Note: tried to add it in types but it messes up the `src` imports. Since stories project is not that important we can leave as it is now and add more types there if necessary even if we duplicate from types folder